src/EventListener/DossierTimelineWorkflowSubscriber.php

```diff
@@ -1,54 +1,54 @@
-<?php

```

Comment:
Hier zijn de codewijzigingen niet leesbaar. Als er geen codewijzigingen zijn, moet dit stuk er uit of anders alleen de diffs tonen.

Reply:
Zie mijn [algemene reactie](https://github.com/Amsterdam/schulddossier/pull/277#issuecomment-1097824167), er is gelukkig een mogelijkheid in GitHub aanwezig om deze whitespace change weg te filteren.